nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java

```diff
@@ -69,15 +70,24 @@ public void setUp() throws Exception {
   public void testPackageUpload() throws IOException
   {
     uploadPackages(MONGO_PKG_FILE_NAME_600_TGZ, MONGO_PKG_FILE_NAME_728_TGZ);
-    //Verify DB contains data about uploaded component and asset
-    Component component = findComponent(repository, MONGO_PKG_NAME);
-    assertThat(component.name(), is(equalTo(MONGO_PKG_NAME)));
-    assertThat(component.version(), is(equalTo(MONGO_PKG_VERSION_600)));
-
-    //Verify Asset is created.
-    Asset asset = findAsset(repository, MONGO_PATH_FULL_600_TARGZ);
-    assertThat(asset.name(), is(equalTo(MONGO_PATH_FULL_600_TARGZ)));
-    assertThat(asset.format(), is(equalTo(HELM_FORMAT_NAME)));
+    //Verify DB contains data about uploaded components and assets
+    Assert.assertTrue(componentAssetTestHelper.componentExists(repository, MONGO_PKG_NAME, MONGO_PKG_VERSION_600));
```

Comment:
Would you mind to use static import for `assertXXX` methods? Also, please remove unused imports.

Reply:
fixed 8402364654feb6ef462e10fada1794abf0a580c6